code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -36,3 +36,7 @@
 		adjustBruteLoss(10)
 	else if(bodytemperature > maxbodytemp)
 		adjustBruteLoss(15)
+
+/mob/living/simple_animal/hostile/retaliate/clown/attack_hand(mob/living/carbon/human/M)

```

Comment:
This happening only on attack_hand and not attackby etc looks weird. Maybe make it generic (something like the hit_reaction system in /human). Just make simple_animals call some proc when they get hit, and call that.